scalecube-organization/Dockerfile

```diff
@@ -5,6 +5,9 @@ ARG SERVICE_NAME
 ARG EXECUTABLE_JAR
 ENV SERVICE_NAME ${SERVICE_NAME}
 
-COPY target/lib /opt/om2/lib
-COPY target/${EXECUTABLE_JAR}.jar /opt/om2/app.jar
-ENTRYPOINT ["/usr/bin/java", "-jar", "-Dlog4j.configurationFile=exchange-log4j2-cloud.xml", "/opt/om2/app.jar"]
\ No newline at end of file
+WORKDIR /opt/om2
+
+COPY target/lib lib
+COPY target/${EXECUTABLE_JAR}.jar app.jar
+
+ENTRYPOINT exec java $JAVA_OPTS -Dlog4j.configurationFile=log4j2.xml -jar app.jar
```

Comment:
No need to specify this `-Dlog4j.configurationFile=log4j2.xml` ;  You do need to specify this when log4j configuration files  differs from default one, e.g. if it's called log4j2-yadayada.xml

Reply:
yeah but for now we don't have `log4j2-yadayada.xml`
and maybe it will be another PR